src/planscape/forsys/parse_forsys_output.py

```diff
@@ -193,8 +193,9 @@ class ForsysScenarioOutput():
     # ---------------------------
     # string patterns for headers
     # ---------------------------
-    # The priority contribution header in the "project output" dataframe.
-    # Weighted priority score is priority weight * priority contribution.
+    # The priority treatment impact header in the forsys "project output"
+    # dataframe.
+    # Weighted priority score is priority weight * priority treatment impact.
```

Comment:
if this term isn't defined anywhere, can you also define it here

Reply:
Which term do you mean?

Reply:
sorry! github comments are weird sometimes - "priority treatment impact"

Reply:
I see.
I tried to describe the field again since I think the original point of confusion was what the field means.